src/sidebar/components/excerpt.js

```diff
@@ -39,12 +52,35 @@ InlineControls.propTypes = {
 
 const noop = () => {};
 
+/**
+ * @typedef ExcerptProps
+ * @prop {Object} [children]
+ * @prop {boolean} [inlineControls] - If `true`, the excerpt provides internal
+ *   controls to expand and collapse the content. If `false`, the caller sets
+ *   the collapsed state via the `collapse` prop.  When using inline controls,
+ *   the excerpt is initially collapsed.
+ * @prop {boolean} [collapse] - If the content should be truncated if its height
+ *   exceeds `collapsedHeight + overflowThreshold`.  This prop is only used if
+ *   `inlineControls` is false.
+ * @prop {number} [collapsedHeight] - Maximum height of the container, in pixels,
```

Comment:
I don't think `collapsedHeight` is really optional from a look through the code. This was a mistake in the old `propTypes` documentation. It looks to me like `children` and `collapsedHeight` should both be required props, but everything else is optional.

Reply:
good catch